File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b, c = map(int, input().split(" "))

if a == b:
    if a == c:
        print(10000 + (1000 * a))
    else:
        print(1000 + (100 * a))
elif b == c:
    if a == c:
        print(10000 + (1000 * b))
    else:
        print(1000 + (100 * c))
elif a == c:
    if a == b:
        print(10000 + (1000 * c))
    else:
        print(1000 + (100 * a))
else:
    print(max(a, b, c) * 100)
        

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.go
package main

import (
    "fmt"
)

func main() {
    var a, b, c int
    
    _, err := fmt.Scanln(&a, &b, &c)
    if err != nil {
        fmt.Println(err)
        return
    }
    
    if a == b && b == c {
        fmt.Println(10000 + (a * 1000))
    } else if (a == b) && a != c {
        fmt.Println(1000 + (a * 100))
    } else if (a == c) && a != b {
        fmt.Println(1000 + (a * 100))
    } else if (b == c) && b != a {
        fmt.Println(1000 + (b * 100))
    } else {
        fmt.Println(max(a, b, c) * 100)
    }
}